packages/fast-usdc/src/cli/operator-commands.js

```diff
@@ -24,17 +48,9 @@ export const addOperatorCommands = program => {
       'after',
       '\nPipe the STDOUT to a file such as accept.json, then use the Agoric CLI to broadcast it:\n  agoric wallet send --offer accept.json --from gov1 --keyring-backend="test"',
     )
-    .option(
-      '--offerId <string>',
-      'Offer id',
-      String,
-      `operatorAccept-${Date.now()}`,
-    )
+    .option('--offerId <string>', 'Offer id', String, `operatorAccept-${now()}`)
     .action(async opts => {
-      const networkConfig = await fetchEnvNetworkConfig({
-        env: process.env,
-        fetch,
-      });
+      const networkConfig = await fetchEnvNetworkConfig({ env, fetch });
```

Comment:
I guess `env` was introduced at some point. It's not great to have both `env` and a config file that can conflict. I'd ask that we at least make https://github.com/Agoric/agoric-sdk/blob/ef82fbf7466ea1b57c321f735d73fcee2c66fd11/packages/fast-usdc/src/cli/transfer.js#L30-L32 use `fetchEnvNetworkConfig` also, and remove [agoricRpc](https://github.com/Agoric/agoric-sdk/blob/ef82fbf7466ea1b57c321f735d73fcee2c66fd11/packages/fast-usdc/src/cli/config.js#L9) from the config file. Also, not sure how well documented the `env` thing is, might be worth adding to a README or help command somehow, even if the only intended user is "us".

Reply:
>I guess env was introduced at some point. 
Reading from `AGORIC_NET` from env has been in the `agoric` command for years. https://github.com/Agoric/agoric-sdk/pull/10570/ gave `getNetworkConfig` a more accurate name of `fetchEnvNetworkConfig`.

> use fetchEnvNetworkConfig also
I support this suggestion and the rationale.

> Also, not sure how well documented the env thing is, might be worth adding to a README or help command somehow, even if the only intended user is "us".

The intended user of `fetchEnvNetworkConfig` is anyone developing a CLI tool that wants to use `AGORIC_NET` env var to pick the network. +1 to adding to this CLI tool that it observes `AGORIC_NET` env var. Though optional on this PR imo

Reply:
Ok. I migrated `transfer` to use AGORIC_NET and added an epilogue to the help text for all commands:

```
$ yarn -s fast-usdc transfer --help
Usage: fast-usdc transfer [options] <amount> <dest>

Transfer USDC from Ethereum/L2 to Cosmos via Fast USDC

Arguments:
  amount      Amount to transfer denominated in uusdc
  dest        Destination address in Cosmos

Options:
  -h, --help  display help for command

  Agoric test networks provide configuration info at, for example,

  https://devnet.agoric.net/network-config

  To use RPC endpoints from such a configuration, use:
  export AGORIC_NET=devnet

  Use AGORIC_NET=local or leave it unset to use localhost and chain id agoriclocal.
  ```